src/Akka.Persistence.Linq2Db.Data.Compatibility.Tests/DataCompatibilitySpecBase.cs

```diff
@@ -134,35 +139,21 @@ public async Task DisposeAsync()
         protected async Task TruncateEventsToLastSnapshot()
         {
             var region = TestCluster!.ShardRegions[0];
-            var cts = new CancellationTokenSource(10.Seconds());
-            try
+            using var cts = new CancellationTokenSource(10.Seconds());
+            foreach (var id in Enumerable.Range(0, 100))
             {
-                var tasks = Enumerable.Range(0, 100).Select(id =>
-                    region.Ask<(string, StateSnapshot?)>(new Truncate(id), cts.Token)).ToList();
-                while (tasks.Count > 0)
-                {
-                    var task = await Task.WhenAny(tasks);
-                    if (cts.Token.IsCancellationRequested)
-                        break;
-
-                    tasks.Remove(task);
-                    var (id, lastSnapshot) = task.Result;
-                    if (lastSnapshot is { })
-                        Output.WriteLine(
-                            $"{id} data truncated. " +
-                            $"Snapshot: [Total: {lastSnapshot.Total}, Persisted: {lastSnapshot.Persisted}]");
-                    else
-                        throw new XunitException($"Failed to truncate events for entity {id}");
-                }
-
-                if (cts.IsCancellationRequested)
-                {
-                    throw new TimeoutException("Failed to truncate all data within 10 seconds");
-                }
+                var (_, lastSnapshot) = await region.Ask<(string, StateSnapshot?)>(new Truncate(id), cts.Token);
+                if (lastSnapshot is { })
```

Comment:
I think I might be too much of a dinosaur, but what does this pattern match mean? Not null?

Reply:
Yes, `myVar is { }` is the new pattern matching way of saying that `myVar is not null` or `myVar != null`